server/controllers/authController/authController.test.js

```diff
@@ -0,0 +1,24 @@
+import AuthController from '.'
+import { createRequestWithUser, createResponse } from '../../../test/fixtures'
+
+describe('AuthController', () => {
+  describe(AuthController.show, () => {
+    describe('When successful', () => {
+      it('returns a status of 200 and a user object', async () => {
+        const request = createRequestWithUser()
+        const response = createResponse()
+
+        await AuthController.show(request, response)
+
+        expect(response.status).toHaveBeenCalledWith(200)
+        expect(response.json).toHaveBeenCalledWith({
+          data: {
+            uid: 'user-uid',
+            display_name: 'Display Name',
+            icon: 'icon',
+          },
+        })
```

Comment:
let's use fixture data here.

Reply:
[a6af4fd](https://github.com/AMP-SCZ/dpdash/pull/317/commits/a6af4fdc187f5764b6233074f05fd8ff2993e74a)

Reply:
we are we creating 2 fixtures for the user we expect to return? Let's create one, `const user = createUser()`.